cmd/rook/object/user.go

```diff
@@ -0,0 +1,331 @@
+
+	if len(users) == 0 {
+		return "", nil
+	}
+
+	var buffer bytes.Buffer
+	w := rook.NewTableWriter(&buffer)
+
+	fmt.Fprintln(w, "USER ID\tDISPLAY NAME\tEMAIL")
+
+	for _, b := range users {
+		fmt.Fprintf(w, "%s\t%s\t%s\n", b.UserID, *b.DisplayName, *b.Email)
+	}
+
+	w.Flush()
+	return buffer.String(), nil
+}
+
+var userGetCmd = &cobra.Command{
+	Use:   "get [User ID]",
+	Short: "Gets object user info",
+}
+
+func getUserEntry(cmd *cobra.Command, args []string) error {
+	rook.SetupLogging()
+
+	if len(args) == 0 {
+		return fmt.Errorf("Missing required argument User ID")
+	}
+
+	if len(args) > 1 {
+		return fmt.Errorf("Too many arguments")
+	}
+
+	if err := flags.VerifyRequiredFlags(cmd, []string{}); err != nil {
+		return err
+	}
+
+	c := rook.NewRookNetworkRestClient()
+	out, err := getUser(c, args[0])
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	fmt.Print(out)
+	return nil
+}
+
+func getUser(c client.RookRestClient, id string) (string, error) {
+	user, err := c.GetObjectUser(id)
+
+	if client.IsHttpNotFound(err) {
+		return "", fmt.Errorf("Unable to find user %s", id)
+	}
+
+	if err != nil {
+		return "", fmt.Errorf("failed to get user: %+v", err)
+	}
+
+	var buffer bytes.Buffer
+	w := rook.NewTableWriter(&buffer)
+
+	fmt.Fprintf(w, "User ID:\t%s\n", user.UserID)
+	fmt.Fprintf(w, "Display Name:\t%s\n", *user.DisplayName)
+	fmt.Fprintf(w, "Email:\t%s\n", *user.Email)
+	fmt.Fprintf(w, "Access Key:\t%s\n", *user.AccessKey)
+	fmt.Fprintf(w, "Secret Key:\t%s\n", *user.SecretKey)
+
+	w.Flush()
+	return buffer.String(), nil
+}
+
+var userDeleteCmd = &cobra.Command{
+	Use:   "delete [User ID]",
+	Short: "Deletes a the object user",
+}
+
+func deleteUserEntry(cmd *cobra.Command, args []string) error {
+	rook.SetupLogging()
+
+	if len(args) == 0 {
+		return fmt.Errorf("Missing required argument User ID")
+	}
+
+	if len(args) > 1 {
+		return fmt.Errorf("Too many arguments")
+	}
+
+	if err := flags.VerifyRequiredFlags(cmd, []string{}); err != nil {
+		return err
+	}
+
+	c := rook.NewRookNetworkRestClient()
+	out, err := deleteUser(c, args[0])
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	fmt.Print(out)
+	return nil
+}
+
+func deleteUser(c client.RookRestClient, id string) (string, error) {
+	err := c.DeleteObjectUser(id)
+
+	if client.IsHttpNotFound(err) {
+		return "", fmt.Errorf("Unable to find user %s", id)
+	}
+
+	if err != nil {
+		return "", fmt.Errorf("failed to delete user: %+v", err)
+	}
+	return "User deleted\n", nil
+}
+
+var userCreateCmd = &cobra.Command{
+	Use:   "create [User ID] [Display Name]",
+	Short: "Creates an object store user",
+}
+
+func createUserEntry(cmd *cobra.Command, args []string) error {
+	rook.SetupLogging()
+
+	if len(args) == 0 {
+		return fmt.Errorf("Missing required arguments User ID and Display Name")
+	}
+
+	if len(args) == 1 {
+		return fmt.Errorf("Missing required argument Display Name")
+	}
+
+	if len(args) > 2 {
+		return fmt.Errorf("Too many arguments")
```

Comment:
instead of positional arguments, our pattern is named arguments that are required. much more readable that way